pkg/controller/resource/daemonset.go

```diff
@@ -56,13 +58,16 @@ func getDaemonSetLists(clientSet *kubernetes.Clientset, ce *chaosTypes.EngineInf
 }
 
 // This will check and count the total chaos enabled application
-func checkForEnabledChaos(targetAppList *v1.DaemonSetList, ce *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, int) {
+func checkForChaosEnabledDaemonSet(targetAppList *v1.DaemonSetList, ce *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, int, error) {
 	chaosEnabledDaemonSet := 0
 	for _, daemonSet := range targetAppList.Items {
 		ce.AppName = daemonSet.ObjectMeta.Name
 		ce.AppUUID = daemonSet.ObjectMeta.UID
 		annotationValue := daemonSet.ObjectMeta.GetAnnotations()[ChaosAnnotationKey]
 		chaosEnabledDaemonSet = CountTotalChaosEnabled(annotationValue, chaosEnabledDaemonSet)
+		if chaosEnabledDaemonSet > 1 {
+			return ce, chaosEnabledDaemonSet, errors.New("too many chaos candidates with same label, either provide unique labels or annotate only desired app for chaos")
```

Comment:
Changes are good @chandankumar4 . Considering we now have separate files for each resource type - can we customize the error messages accordingly too? This might give better user experience. 

Ex: "too many deployments with specified label are annotated for chaos, either provide unique labels or annotate only desired app for chaos"

Reply:
Done